server/src/main/java/com/thoughtworks/go/config/update/PluginProfileCommand.java

```diff
@@ -102,4 +102,8 @@ private String getTagName() {
 
     protected abstract boolean isAuthorized();
 
+    public boolean isNotADeleteCommand() {
```

Comment:
I think this should be renamed to `shouldValidatePluginProperties` as that is what is being checked.